aten/src/ATen/Context.h

```diff
@@ -86,4 +87,8 @@ static inline bool hasCUDA() {
   return globalContext().hasCUDA();
 }
 
+static inline int64_t current_device() {
```

Comment:
My thinking is that we want the C++ function names to match their Python equivalents. Since we use `snake_case` in Python, we should prefer `snake_case` in C++.

It would be nice to make the naming completely consistent within ATen, but that seems low priority.

@zdevito if you have any naming preferences.

Reply:
Sure, I don't really have a strong preference as long as they are eventually consistent.